react/QuantitySelector.tsx

```diff
@@ -20,16 +20,26 @@ export type QuantitySelectorStepType = 'unitMultiplier' | 'singleUnit'
 interface Props {
   mode?: QuantitySelectorMode
   quantitySelectorStep?: QuantitySelectorStepType
+  IncreaseIcon?: React.FC
+  DecreaseIcon?: React.FC
```

Comment:
I prefer `ReactElement` and if really needs children, you can add `PropsWithChildren`
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L146
```suggestion
  IncreaseIcon?: React.ReactElement
  DecreaseIcon?: React.ReactElement
```

Reply:
I try this types, but both cause "Dont have call signatures" error.